pkg/apis/tenancy/v1alpha1/types.go

```diff
@@ -102,21 +102,143 @@ type ClusterWorkspaceType struct {
 
 	// +optional
 	Spec ClusterWorkspaceTypeSpec `json:"spec,omitempty"`
+
+	// +optional
+	Status ClusterWorkspaceTypeStatus `json:"status"`
 }
 
+const (
+	// ClusterWorkspaceCreatorUserLabel is the name of the user creating this ClusterWorkspace.
+	// It is used by the initialization virtual workspace to authorize object creation. It is applied during
+	// admission, removed after initialization and is not exposed to the user in Workspaces.
+	ClusterWorkspaceCreatorUserLabel = "tenancy.kcp.dev/creator-user"
+	// ClusterWorkspaceCreatorGroupsLabel is a comma separated list of group of the user creating
+	// this ClusterWorkspace. It is used by the initialization virtual workspace to authorize object creation
+	// It is applied during admission, removed after initialization and is not exposed to the
+	// user in Workspaces.
+	ClusterWorkspaceCreatorGroupsLabel = "tenancy.kcp.dev/creator-groups"
+)
+
 type ClusterWorkspaceTypeSpec struct {
-	// initializers are set of a ClusterWorkspace on creation and must be
+	// bootstrapManifests references a set of Kubernetes manifests that are applied to the
+	// workspace on initialisation.
+	//
+	// If a workspace type extends on at a higher level, the bootstrap manifests of
+	// the higher level are applied first, then those of the lower levels.
+	//
+	// Every bootstrap manifest created is authorized against the user creating the
+	// workspace.
+	//
+	// +optional
+	BootstrapManifests *BootstrapManifests `json:"bootstrapManifests,omitempty"`
+
+	// initializers are set on a ClusterWorkspace on creation and must be
 	// cleared by a controller before the workspace can be used. The workspace
 	// will stay in the phase "Initializing" state until all initializers are cleared.
 	//
+	// A controller responsible for initializing workspaces of a given type must
+	// be granted verb="initialize" permissions on the ClusterWorkspaceType resource.
+	// These controllers are not able to access a workspace at any other phase.
+	//
+	// If a workspace type extends one at a high level the lists of initializers
```

Comment:
nit: "extends one at a high level" is a bit hard to read/grok

Reply:
compare the new wording. Am open for something better.